src/views/blocks/rsvp/form.php

```diff
@@ -8,30 +8,34 @@
  *
  * See more documentation about our Blocks Editor templating system.
  *
- * @link {INSERT_ARTICLE_LINK_HERE}
+ * @link    {INSERT_ARTICLE_LINK_HERE}
  *
- * @since 4.9
- * @since 4.12.0 Add $post_id to filter for hiding opt-outs.
+ * @since   4.9
+ * @since   4.12.0 Add $post_id to filter for hiding opt-outs.
+ * @since   TBD Add vars to docblock.
  *
  * @version 4.12.0
  *
+ * @var Tribe__Tickets__Editor__Template $template
+ * @var Tribe__Tickets__RSVP             $rsvp
+ * @var int                              $post_id
  */
 
 $going      = $this->get( 'going' );
-$must_login = ! is_user_logged_in() && tribe( 'tickets.rsvp' )->login_required();
+$must_login = ! is_user_logged_in() && $rsvp->login_required();
 ?>
 <!-- This div is where the AJAX returns the form -->
 <div class="tribe-block__rsvp__form">
 	<?php if ( ! empty( $going ) && ! $must_login ) :
 		$ticket = $this->get( 'ticket' );
-		$args = [
+		$args   = [
 			'ticket_id' => $ticket->ID,
 			'post_id'   => $post_id,
-			'ticket'    => tribe( 'tickets.rsvp' )->get_ticket( get_the_id(), $ticket->ID ),
+			'ticket'    => $rsvp->get_ticket( get_the_id(), $ticket->ID ),
 			'going'     => esc_html( $going ),
 		];
 
 		// can't escape, contains html
-		echo tribe( 'tickets.editor.template' )->template( 'blocks/rsvp/form/form', $args, false );
+		echo $template->template( 'blocks/rsvp/form/form', $args, false );
```

Comment:
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$template'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$template'.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$args'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$args'.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'false'

<sub>hash: 597a070c9c8eed91c055ebe9dbcf2cb6</sub>

Reply:
Might need to be `$this`

Reply:
correct